pages/k8s/vsphere-integration.md

```diff
@@ -42,23 +48,31 @@ juju model-config datastore=mydatastore primary-network=mynetwork
 ### Installing
 
 If you install **Charmed Kubernetes** [using the Juju bundle][install],
-you can add the vsphere-integrator at the same time by using the following
+you can add both `vsphere-cloud-provider` and `vsphere-integrator` at the same time by using the following
 overlay file ([download it here][asset-vsphere-overlay]):
 
 ```yaml
 description: Charmed Kubernetes overlay to add native vSphere support.
 applications:
+  kubernetes-control-plane:
+    options:
+      allow-privileged: "true"
   vsphere-integrator:
-    annotations:
-      gui-x: "600"
-      gui-y: "300"
     charm: vsphere-integrator
     num_units: 1
     trust: true
+  vsphere-cloud-provider:
+    charm: vsphere-cloud-provider
 relations:
-  - ['vsphere-integrator', 'kubernetes-control-plane']
-  - ['vsphere-integrator', 'kubernetes-worker']
-  ```
+- - vsphere-cloud-provider:certificates
+  - easyrsa:client
+- - vsphere-cloud-provider:kube-control
+  - kubernetes-control-plane:kube-control
+- - vsphere-cloud-provider:external-cloud-provider
+  - kubernetes-control-plane:external-cloud-provider
+- - vsphere-cloud-provider:vsphere-integration
+  - vsphere-integrator:clients
+```
```

Comment:
@mateoflorido should there be an inclusion of a section regarding `Upgrading`.  WHat's the process look like for doing an upgrade of a vSphere cloud from 1.25->1.26

* I think the charms would need to be upgraded
* Then the vsphere-cloud-provider should be installed and related 
* Do the storage volumes need to be migrated from:
    * `kubernetes.io/vsphere-volume` -> `csi.vsphere.vmware.com`
* Then remove the relation of the integrator -> worker and control-pane
* NOW the snap channel can be changed from 1.25 -> 1.26

Does this need to be a separate task from just updating the installation docs?

Reply:
Yes, I believe the scope of this PR is limited to updating the installation instructions. I'll be sure that we include your suggestions in another PR soon.